sources/osgShadow/shaders/hash.glsl

```diff
@@ -1,24 +1,52 @@
 //// hash glsl
 
-//note: value noise
-float hashNormalized11( const in float n )
+// Dave Hoskins: hash without sin
+//https://www.shadertoy.com/view/4djSRW
+#define MOD2 vec2(443.8975,397.2973)
+//----------------------------------------------------------------------------------------
+//  1 out, 1 in...
+//note: normalized uniform random, [0;1[
+float hash11(const in float p)
+{
+  vec2 p2 = fract(vec2(p) * MOD2);
```

Comment:
this file use 2 spaces for indentation but this one https://github.com/cedricpinson/osgjs/pull/411/files#diff-f88a75996357a297e59617395fc78b26R8 use 4 spaces. Looks like we have the same problem on osgjs, could you use 4 spaces for all glsl files plz ?